docs/examples/control_ish_ufschem-example.yaml

```diff
@@ -0,0 +1,294 @@
+# General Description:
+# - Any key that is specific for a plot type will begin with `ts` for timeseries, `ty` for taylor.
+# - Some keys/groups are optional.
+# - For now, all plots except time series average over the analysis window.
+# - Setting axis values
+#   - If set_axis = True in data_proc section of each plot_grp,
+#     the yaxis for the plot will be set based on the values
+#     specified in the obs section for each variable.
+#   - If set_axis is set to False, then defaults will be used.
+#   - 'vmin_plot' and 'vmax_plot' are needed for
+#     'timeseries', 'spatial_overlay', and 'boxplot'.
+#   - 'vdiff_plot' is needed for 'spatial_bias' plots
+#   - 'ty_scale' is needed for 'taylor' plots.
+#   - 'nlevels' or the number of levels used in the contour plot can also optionally be provided for spatial_overlay plot.
+#   - If set_axis = True and the proper limits are not provided in the obs section,
+#     a warning will print, and the plot will be created using the default limits.
+
+analysis:
+  start_time: "2017-07-01-00:00:00" # UTC
+  end_time: "2017-07-03-00:00:00" # UTC
+  output_dir: ./output/ish_ufschem  # relative to the program using this control file
+  #   Currently, the directory must exist or plot saving will error and fail.
+  debug: True
+
+model:
+  ufschem_v1: # model label
+    files: example:ufschem:2017-07
+    mod_type: "ufschem" #change
+    mod_kwargs:
+      surf_only: True  # specify that we have only one vertical level; WRF-Chem specific
+    radius_of_influence: 100000 # meters
+
+    extra_calc: # Add more complex calculations and their associated variables here.  
+
+    ####### Extra_calc naming example
+    # variable_calculating
+      # variable required for calculation: "variable name in model"
+      
+      dewpoint:
+        specific_hum: "spfh" # string of what it is in the model 
+      rel_hum: 
+        specific_hum: "spfh"
+      windspeed:
+        u_comp: "ugrd"  
+        v_comp: "vgrd" 
+      winddir: 
+        u_comp: "ugrd" 
+        v_comp: "vgrd" 
+        
+    mapping: # of _model_ species name to _obs_ species name
+      ish: # specifically for the obs labeled 'ish'
+        temperature_k: "t"
+        windspeed: "ws"
+        dewpoint: "dpt"
+
+    variables: 
+      winddir: None # provide this as a none, otherwise YAML will error.
+      
+     # if you do not want plots specifically for wind direction, simply add it to the variable list here. 
+     # Wind direction is needed if you wish to plot wind roses. 
+      
+    projection: ~
+    plot_kwargs: # optional
+      color: "magenta"
+      marker: "s"
+      linestyle: "-"
+      
+obs:
+  ish: # obs label
+    use_ish: True
```

Comment:
You don't need this. We had use_airnow in the beginning but it's not actually used in the code anywhere. So for clarity I would remove it unless you did add it in to do something.